app/models/repaymentHistory/RepaymentHistoryUtils.scala

```diff
@@ -125,13 +134,13 @@ object RepaymentHistoryUtils {
     )
   }
 
-  private def creditEntry(payment: Payment, isAgent: Boolean)(implicit messages: Messages, dateServiceInterface: DateServiceInterface): PaymentHistoryEntry = {
+  private def creditEntry(payment: Payment, isAgent: Boolean, isPoaReconciliationCredit: Boolean = false)(implicit messages: Messages, dateServiceInterface: DateServiceInterface): PaymentHistoryEntry = {
```

Comment:
more refactoring to be done as a part of : https://jira.tools.tax.service.gov.uk/browse/MISUV-8654